### PR TITLE
Slides 1-3.

### PR DESCRIPTION
**Pull request by jonthegeek**

### CODE REVIEW

File: 03-data_visualisation.Rmd [hunk 1]
```diff
@@ -1,14 +1,164 @@
...
+  - shape
+  - color
+  - alpha (transparency)
+  - x
+  - y
+  - others, sometimes specific to a geom.
+- Use the `mapping` argument and the `aes` function to map an aesthetic to a variable in `data`.
+- Or assign the value outside of `mapping` to set it for everything:
+  - `geom_point(mapping = aes(x = displ, y = hwy), color = "blue")`
+  - A super common error: Trying to do that *inside* the `aes` call. Results in random effects because `aes()` automatically figures out the necessary **scale** for your data.
```
Comment (jonathanbratt) [inline 03-data_visualisation.Rmd:65]: Trying to do what? The color?

**Review by jonathanbratt — APPROVED** [review]
noted ambiguous "that", but since this is going to be presented live it will probably be fine.

File: 03-data_visualisation.Rmd (hunk 1 above)
Comment (jonathanbratt) [reply 03-data_visualisation.Rmd:65]: actually, on a second look, I don't think it's that ambiguous. So never mind. :)

Comment (jonthegeek) [reply 03-data_visualisation.Rmd:65]: The "that" is "this thing that's in this family of bullets." I *think* it's clear-enough to present.

**Pull request merged**